skimage/data/_registry.py

```diff
@@ -121,6 +121,8 @@
     "restoration/tests/astronaut_rl.npy":
         "3f8373e2c6182a89366e51cef6624e3625deac75fdda1079cbdad2a33322152c",
     "restoration/tests/camera_rl.npy": "fd4f59af84dd471fbbe79ee70c1b7e68a69864c461f0db5ac587e7975363f78f",
+    # "restoration/tests/camera_dnp.npy":
+        # "05d6831f9c81f83fa820237052e894259f1a8f0ae2974449f60195439d369338",
```

Comment:
To add sample data, please create a PR to gitlab.com/scikit-image/data (search for "pooch" in our [contributing guide](https://scikit-image.org/docs/dev/contribute.html)). You can see an example for doing this here:

https://github.com/scikit-image/scikit-image/pull/4922/files

Reply:
Just placed in a merge request on Gitlab. waiting for that one to merge and then I continue.

Reply:
Test still commented out, wait for merge on gitlab